specs/DesignDiscussions/InputLabels.md

```diff
@@ -72,17 +72,55 @@ This conflict speaks to a general tension inherent in the design of Adaptive Car
 
 In order to gather more data to help us balance the needs of the card authors and hosts, we will send a survey to customers who have expressed an interest in improving out input story. The following options are under consideration, and will be evaluated against survey data to determine which if any will be included in our v1 implementation.
 
+> ## Rendering for string labels
```

Comment:
Are these the options described above for Formatting? If so these need to be subheadings under Formatting (###?)